yivi_core/assets/locales/en.json

```diff
@@ -648,6 +648,63 @@
     "hint_state_on": "Double tap to turn off",
     "hint_state_off": "Double tap to turn on"
   },
+  "driving_licence": {
+    "scan": {
+      "title": "Scan driving licence",
+      "success": "Success",
+      "success_explanation": "The driving licence has been successfully scanned.",
+      "manual": "Enter manually",
+      "error": "Could not read driving licence. Please try again."
+    },
+    "manual": {
+      "title": "Enter driving licence details",
+      "explanation": "We need the Machine readable zone (MRZ) of your driving licence to access the NFC chip of the document.",
```

Comment:
The Machine readable zone (MRZ) of your driving license is needed to access the RFID-chip of the document.